docs/usage.rst

```diff
@@ -167,3 +167,14 @@ Examples
     New passphrase: 
     Enter same passphrase again: 
     Key file "/home/USER/.attic/keys/tmp_encrypted_repo" updated
+
+.. include:: usage/serve.rst.inc
+
+Examples
+~~~~~~~~
+::
+
+    # Allow an SSH keypair to only run attic, and only have access to repo.attic
+    # This will help to secure an automated remote backup system.
+    $ cat ~/.ssh/authorized_keys
+    command="attic serve --restrict-to-path /path/to/repo.attic" ssh-rsa AAAAB3[...]
```

Comment:
I was actually wondering about that. I was struggling to keep everything
under 80 columns wide. Is that a relevant style restriction for this
project? And if so, how should I wrap the text in the examples? With a
'\' like in the shell?

On Thu, Apr 09, 2015 at 03:44:45AM -0700, TW wrote:
> > @@ -167,3 +167,14 @@ Examples
> >      New passphrase: 
> >      Enter same passphrase again: 
> >      Key file "/home/USER/.attic/keys/tmp_encrypted_repo" updated
> > +
> > +.. include:: usage/serve.rst.inc
> > +
> > +Examples
> > +~~~~~~~~
> > +::
> > +
> > +    # Allow an SSH keypair to only run attic, and only have access to repo.attic
> > +    # This will help to secure an automated remote backup system.
> > +    $ cat ~/.ssh/authorized_keys
> > +    command="attic serve --restrict-to-path /path/to/repo.attic" ssh-rsa AAAAB3[...]
> 
> maybe use the same repository path in quickstart.rst and in usage.rst (and see elsewhere, what is usually used).
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/jborg/attic/pull/277/files#r28051664